nbtools/run_notebook.py

```diff
@@ -340,6 +348,11 @@ def run_notebook(path, inputs=None, outputs=None, inputs_pos=1, replace_inputs_p
         # Prepare execution results: execution state, notebook outputs and error info (if exists)
         if failed:
             exec_res = {'failed': failed, 'failed cell number': error_cell_num, 'traceback': traceback_message}
+
+            # Re-raise exception if needed
+            if raise_exception:
+                returned_value.put(exec_res)
```

Comment:
Please, rename `returned_value` to something `queue` related. `_queue`, `_output_queue`,